=== System Information Report ===
>>> Generated on: Mon Jan  6 08:13:50 AM UTC 2025
==========================
>>> CPU Architecture: x86_64
>>> CPU Byte Order: Little Endian
>>> CPU Cores: 16
>>> CPU Model: AMD Ryzen 7 PRO 8700GE w/ Radeon 780M Graphics
>>> CPU Cache Information:
L1d Cache: 256 KiB (8 instances)
L1i Cache: 256 KiB (8 instances)
L2 Cache: 8 MiB (8 instances)
L3 Cache: 16 MiB (1 instance)
>>> RAM Size: 61Gi
>>> Hard Disk Information:
/dev/md2 : 437G total, 201G free
/dev/md1 : 989M total, 693M free
/dev/nvme2n1p1 : 256M total, 256M free
/dev/nvme1n1 : 1.9T total, 740G free
==========================
=== Nimbus-ETH1 Benchmarking Report ===
>>> Total time spent in benchmarking (nanoseconds): 555445004857
>>> Total time spent in benchmarking (human readable format): 9 minutes 15 seconds
>>> Start block number is 20000001
>>> End block number is 20005001
>>> Benchmarking was run using /home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/build/nimbus_execution_client import \
    --network=mainnet \
    --data-dir='/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/data/shared_mainnet_0' \
    --era1-dir=/data/era1 \
    --era-dir=/data/era \
    --log-level=INFO \
    --max-blocks=5000 \
    --debug-csv-stats=/home/data/nimbus-eth1-benchmarks/blocks-import-benchmark.csv
==========================
=== Comparison of last two benchmarks ===
python -m venv /home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/stats
source /home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/stats/bin/activate
pip install -r /home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/scripts/requirements.txt >/dev/null 2>&1
python /home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/scripts/block-import-stats.py /home/data/nimbus-eth1-benchmarks/short-benchmark/20250106T081350_0ffc17d1/blocks-import-benchmark.csv /home/data/nimbus-eth1-benchmarks/short-benchmark/20250106T075839_0ffc17d1/blocks-import-benchmark.csv 2>&1 || true
Traceback (most recent call last):
  File "/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/scripts/block-import-stats.py", line 198, in <module>
    main()
  File "/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/scripts/block-import-stats.py", line 131, in main
    df = df.interpolate(method="index").reindex(contender.index)
  File "/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/stats/lib/python3.10/site-packages/pandas/core/frame.py", line 5378, in reindex
    return super().reindex(
  File "/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/stats/lib/python3.10/site-packages/pandas/core/generic.py", line 5610, in reindex
    return self._reindex_axes(
  File "/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/stats/lib/python3.10/site-packages/pandas/core/generic.py", line 5633, in _reindex_axes
    new_index, indexer = ax.reindex(
  File "/home/data/nimbus-eth1-mainnet-master-short-benchmark/repo/stats/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 4429, in reindex
    raise ValueError("cannot reindex on an axis with duplicate labels")
ValueError: cannot reindex on an axis with duplicate labels
deactivate
==========================
